mirrord/sip/src/lib.rs

```diff
@@ -487,7 +487,17 @@ mod main {
     }
 
     /// SIP check for binaries.
-    fn is_binary_sip(path: &Path, data: &[u8], patch_binaries: &[String]) -> Result<bool> {
+    fn is_binary_sip(
+        path: &Path,
+        data: &[u8],
+        patch_binaries: &[String],
```

Comment:
I would concider making a custom struct for patch and skip because it's easy to accidentaly pass them the wrong way around, if they were of different types it would be ok but they are of the same type

Reply:
Thanks for taking a look!
Something like?
```rust
struct SipOptions<'a>  {
    patch: &'a [String],
    skip: &'a [String],
}
```
Let me know if I misunderstand your suggestion.

Reply:
Yeah exactly, just to give names to the arrays

Reply:
Added an option type which wraps `patch` and `skip`, thanks!